crates/ruma-identifiers/src/lib.rs

```diff
@@ -8,11 +8,14 @@
 #![allow(unused_qualifications)]
 #![cfg_attr(docsrs, feature(doc_auto_cfg))]
 
-// Renamed in `Cargo.toml` so we can have a serde feature.
-// Rename it back here because `serde1` is ugly.
+// Renamed in `Cargo.toml` so we can features with the same name as the package.
+// Rename them back here because the `Cargo.toml` names are ugly.
```

Comment:
```suggestion
// Renamed in `Cargo.toml` so we can have features with the same name as the package.
// Rename them back here because the `Cargo.toml` names are ugly.
```